src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -110,16 +111,14 @@ std::string Condition::toJson()
 
   jh.addRaw("attributes", vectorToJson(this->attributes));
 
-  {
-    JsonHelper jhe;
-
-    jhe.addString("q", this->expression.q);
-    jhe.addString("geometry", this->expression.geometry);
-    jhe.addString("coords", this->expression.coords);
-    jhe.addString("georel", this->expression.georel);
+  JsonHelper jhe;
 
-    jh.addRaw("expression", jhe.str());
-  }
+  jhe.addString("q", this->expression.q);
+  jhe.addString("geometry", this->expression.geometry);
+  jhe.addString("coords", this->expression.coords);
+  jhe.addString("georel", this->expression.georel);
+  
+  jh.addRaw("expression", jhe.str());
```

Comment:
Yeah, almost ...
I removed an unnecessary block also - that's why the indentation changed